app/javascript/react/components/App.js

```diff
@@ -1,15 +1,15 @@
-import React from "react";
-import { ApolloProvider } from "@apollo/client";
-import ApolloClient from "../../api/ApolloClient";
-import ApiDemo from "./ApiDemo";
+import React from 'react';
+import { ApolloProvider } from '@apollo/client';
+import ApolloClient from '../../api/ApolloClient';
+import MainComponent from './MainComponent/MainComponent';
+import ApiDemo from './ApiDemo/ApiDemo';
```

Comment:
is our lint rule single quotes? or double quotes?

my preference is double quotes everywhere but in general most lint setups would use single unless you need double? at least in ruby

also we should probably consider having a common root for our controllers so as NOT to have to do all the ../../ referencing using something like

in config/webpacker.yml

```
alias:
  react: app/javascript/
```

so you could do `import ApolloClient from 'api/ApolloClient';`

Reply:
webpack alias added for api